src/_base/helm/app/templates/application/webapp/deployment.yaml

```diff
@@ -1,22 +1,22 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Values.resourcePrefix }}php-fpm
+  name: {{ .Values.resourcePrefix }}webapp
   labels:
-    app.service: {{ .Values.resourcePrefix }}php-fpm
+    app.service: {{ .Values.resourcePrefix }}webapp
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app.service: {{ .Values.resourcePrefix }}php-fpm
+      app.service: {{ .Values.resourcePrefix }}webapp
   template:
     metadata:
       labels:
-        app.service: {{ .Values.resourcePrefix }}php-fpm
+        app.service: {{ .Values.resourcePrefix }}webapp
     spec:
       {{- if not (eq "" (include "application.volumes.wwwDataPaths" .)) }}
       initContainers:
-      - name: php-fpm-volume-permissions
+      - name: {{ .Values.resourcePrefix }}webapp-volume-permissions
```

Comment:
for container names we can get away without the resourcePrefix, in the case of multiple containers it makes knowing which container to specify a bit more consistent

Reply:
removed here now